src/SpeciesCard.js

```diff
@@ -1,13 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import EbiSpeciesIcon from 'react-ebi-species'
 
-const SpeciesCard = () => {
+const SpeciesCard = ({iconSrc, description, content}) => {
   return (
-    <p>SpeciesCard content</p>
+    <div className={`column column-block text-center combo card`} style={{marginBottom:0, paddingBottom: `25px`}}>
+      <span className={`species-icon`} style={{fontSize: `600%`}}>
```

Comment:
If you’re using `rem`s elsewhere I’d use them here as well.

Reply:
Good spot, will make units consistent